packages/foam-cli/package.json

```diff
@@ -11,17 +11,23 @@
     "@oclif/command": "^1",
     "@oclif/config": "^1",
     "@oclif/plugin-help": "^3",
```

Comment:
We need to add foam-core into dependencies as well for npm publishing

Reply:
done!